examples/03-http-any.php

```diff
@@ -29,9 +29,7 @@ function (Psr\Http\Message\ResponseInterface $response) use ($url) {
 $promise->then(
     function ($url) {
         echo 'First successful URL is ' . $url . PHP_EOL;
-    },
-    function ($e) {
-        echo 'An error occurred: ' . $e->getMessage() . PHP_EOL;
+    }, function (Exception $e) {
+        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
Nit: PSR-12 (see above)

```suggestion
    },
    function (Exception $e) {
        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Reply:
Answer: changed (see above)